Services/Infrastructure/OJS.Services.Infrastructure/Constants/CacheConstants.cs

```diff
@@ -15,6 +15,7 @@ public static class CacheConstants
 
         public const string SubmissionsCountByMonthsForPastElevenMonthsKey = "Submissions_count_by_month_for_past_eleven_months";
         public const string SubmissionsCountForLastMonthKey = "Submissions_count_by_month_for_last_month";
+        public const string GetTotalSubmissionsCount = "GetTotalSubmissionsCount";
```

Comment:
The name is not right for a constant.  There should be no verb like "get" there.